src/allmydata/scripts/common.py

```diff
@@ -270,6 +273,20 @@ def get_alias(aliases, path_unicode, default):
     return uri.from_string_dirnode(aliases[alias]).to_string(), path[colon+1:]
 
 def escape_path(path):
-    # this always returns bytes, specifically US-ASCII, valid URL characters
```

Comment:
In 09f3e3f6b, I indicated that the comment is superseded by the docstring, which says `    Return path quoted to US-ASCII, valid URL characters.` More importantly, the (passing) doctest shows clearly what types are expected/allowed.

Reply:
I don't think Tahoe runs doctests as part of CI at the moment.

Reply:
Indeed there is no doctest-running CI configuration and Tahoe has no policy or convention of using doctest to test anything.

It's probably fine to leave this, at worst it will be wrong - which is also the worst case for every single other docstring.